projects/core/src/occ/adapters/cart/index.ts

```diff
@@ -3,3 +3,4 @@ export * from './converters/index';
 export * from './occ-cart-entry.adapter';
 export * from './occ-cart-voucher.adapter';
 export * from './occ-cart.adapter';
+export * from './custom.encoder';
```

Comment:
Can we not export the CustomEncoder, please? It has bad name.
Ticket to improve it later: https://github.com/SAP/spartacus/issues/8658